frontend/.gitignore

```diff
@@ -24,3 +25,4 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 .idea
+.yarn
```

Comment:
Both branches `modularization/frontend` and `itsmo-amzn:merge-copy` still have `word-wrap` version "1.2.3" 
(#585 for the reference).
I think we should fix it after the merge. Wdyt? @maryamkhidir , @itsmo-amzn

Reply:
Yes, we can do that, the #585 PR was closed because it was pointing to main. I can add that once the merge is done.